prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -269,6 +272,27 @@ class ReportFunction : Logging {
         return responseBuilder.build()
     }
 
+    private fun routeToFHIREngine(blobInfo: BlobAccess.BlobInfo, sender: Sender, queue: QueueAccess) {
+        try {
+            if (System.getenv("ENABLE_FHIR_ENGINE")?.isNotEmpty() ?: false)
```

Comment:
Acts as a feature flag defaulted to `off`. Failures should not affect the main pipeline in any way.